### ISSUE TITLE
Update and Save Config Functionality in the GUI

### ISSUE THREAD

**Issue by bnasko** [jhuckaby/Cronicle#377]
First, I want to thank everyone involved in this project. You guys are phenomenal.

I searched the repo looking for an easy way to update my cronicle version but didn't find anything. Is there a linux (ubuntu) command that I could run to update the version without messing up all my jobs/settings?

Alternatively, it would be ideal to have a button placed in the GUI that would allow for updating to the latest version along with another function to backup all settings/jobs as an xml/json file.

**Comment by mikeTWC1984** [comment]
Yeah, there is a cli tool (in the box) to upgrade/export/import data
https://github.com/jhuckaby/Cronicle#upgrading-cronicle
I'm working on a fork with a feature that allows to import/export metadata from GUI, I agree it's pretty useful. I'll publish it sometime soon.  Regarding upgrades - I'm proponent of using docker for such things.I'll create an image and publish some sample too.

**Comment by bnasko** [comment]
My bad, I missed the section on upgrading using the cli tool. Thanks for pointing that out. I'll give it a try tonight as I am already a couple of releases behind.

**Comment by lettore** [comment]
The cli command is not working in a Docker container as it stops the program causing docker stopping the container.
A backup command compatible with Docker is needed.
